src/renderer/features/fellowship-voting-history/model/votes.ts

```diff
@@ -0,0 +1,38 @@
+import { combine, sample } from 'effector';
+import { createGate } from 'effector-react';
+import { or } from 'patronum';
+
+import { attachToFeatureInput } from '@/shared/effector';
+import { nonNullable } from '@/shared/lib/utils';
+import { type ReferendumId } from '@/shared/pallet/referenda';
+import { collectiveDomain } from '@/domains/collectives';
+
+import { fellowshipModel } from './fellowship';
+import { votingHistoryFeatureStatus } from './status';
+
+const gate = createGate<{ referendumId: ReferendumId | null }>({ defaultState: { referendumId: null } });
+
+const $votes = fellowshipModel.$store.map(store => store?.votes ?? []);
+
+const $votesList = combine($votes, gate.state, (votes, { referendumId }) => {
+  if (!votes || referendumId === null) return [];
+
+  return votes[referendumId] ?? [];
+});
+
+const votesUpdate = attachToFeatureInput(votingHistoryFeatureStatus, gate.state);
+
+sample({
+  clock: votesUpdate,
```

Comment:
I think it should be just `gate.open`, it's ok because feature works only when view is shown